### DISCUSSION TITLE
Vaultwarden crashed then all stuff lost.

### DISCUSSION THREAD

**Discussion by SirObby** [dani-garcia/vaultwarden#1769] [Q&A]
### Subject of the issue
For some reason Vaultwarden had exited with a weird code when I started my PC and then I had seen the grayed out Bitwarden Icon in the extension.

### Deployment environment

* vaultwarden version: 1.21.0

* Install method: Docker Image

* Clients used: Web Vault and Extension

* Other relevant details:

### Steps to reproduce
Maybe just crash your PC or get Vaultwarden to somehow crash.

### Expected behaviour
Vaultwarden boots back up fine and the db is saved

### Actual behaviour
Vaultwarden boots up and basically with a fresh database

### Troubleshooting data
I dont think I have any data to back me up.

**Comment by BlackDex** [comment]
With this we can't do anything.
The only thing i can imagine is that you could have configured docker in the wrong way and that it didn't used a volume to store the data.

**Comment by FLX-0x00** [comment]
![image](https://user-images.githubusercontent.com/26464774/121907221-23ee3180-cd2c-11eb-80db-473826a6f44f.png)
